packages/grafana-ui/src/components/Layout/Layout.story.tsx

```diff
@@ -15,31 +15,59 @@ export default {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['orientation'],
+    },
```

Comment:
Shouldn't we disable knobs too or are we disabling all knobs when everything is migrated?
```suggestion
    controls: {
      exclude: ['orientation'],
    },
    knobs: {
      disabled: true,
    },
```

Reply:
good catch Hugo! Thanks!